core/platform/protobuf/gce_protobuf_services.py

```diff
@@ -0,0 +1,72 @@
+# coding: utf-8
+#
+# Copyright 2020 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"Functions and class definitions related to protobuf files used in Oppia-ml"
+
+from core.platform.protobuf.protofiles import training_job_data_pb2 # pylint: disable=no-name-in-module
+
+class TrainingJobResultMessage(object):
+    """TrainingJobDataMessage used to transfer trained classifier and relevant
+    job data to Oppia."""
```

Comment:
Move """ to the next line. (This is standard for multi-line docstrings.)

Reply:
Done.